api/src/main/java/org/openmrs/module/sync2/api/mapper/EventConfigurationMapper.java

```diff
@@ -0,0 +1,9 @@
+package org.openmrs.module.sync2.api.mapper;
+
+import org.openmrs.module.atomfeed.api.model.FeedConfiguration;
+import org.openmrs.module.sync2.api.model.configuration.EventConfiguration;
+
+public interface EventConfigurationMapper {
+
+	EventConfiguration map(FeedConfiguration feedConfiguration);
```

Comment:
This interface should not depend on org.openmrs.module.atomfeed.api.model.FeedConfiguration. Maybe use <T>.

Reply:
Good point, thanks.